aws/vars.tf

```diff
@@ -15,6 +15,12 @@ variable "ha" {
   default     = false
 }
 
+variable "instance_name_prefix" {
```

Comment:
I'm not really sure about the purpose of `instance_name_prefix`. Perhaps we need a way of overriding the `deployment_id` so that it isn't always UUID-timestamp - so if a user wanted to put their project codename or something then they could.

Reply:
I agree that these two variables are too similar. I'll change the name of this variable to `deployment_prefix` and have it be empty by default. Then I'll update the comment to know that that will override `deployment_id` if set, and that `deployment_id` already has a unique default value. I'll also switch all current uses of `instance_name_prefix` to `deployment.id`.